docs/deployment/linux.md

```diff
@@ -0,0 +1,129 @@
+
+Enter the directory using `cd folder_name`.
+
+## Downloading Velocity
+
+You can find the latest version [on the downloads site](https://www.velocitypowered.com/downloads). Download the latest version to your newly created folder (or use FTP to upload it).
+
+As an example, I'll use this command:
+```shell script
+wget -O proxy.jar https://ci.velocitypowered.com/job/velocity-1.1.0/lastSuccessfulBuild/artifact/proxy/build/libs/velocity-proxy-1.1.0-SNAPSHOT-all.jar
+```
+
+## Running Velocity
+
+### The Script
+
+To easily start and stop Velocity, we will use a simple shell script.
+
+To start creating this script, use `nano start.sh`, and paste this in:
+
+```sh
+#!/bin/bash
+PROXY_JAR=proxy.jar
+RAM=512M
+
+while true
+do
+    java -XX:+UseG1GC -XX:G1HeapRegionSize=4M -XX:+UnlockExperimentalVMOptions -XX:+ParallelRefProcEnabled -XX:+AlwaysPreTouch -Xmx${RAM} -Xms${RAM} -jar ${PROXY_JAR}
+    echo "Velocity exited."
+    echo "Starting in 3... (Press [CTRL+C] to stop)"
+    sleep 1
+    echo "Starting in 2... (Press [CTRL+C] to stop)"
+    sleep 1
+    echo "Starting in 1... (Press [CTRL+C] to stop)"
+    sleep 1
+done
+```
+
+[_Note: This uses Tux's recommended HotSpot Java flags as of 04/25/2020.]( https://discord.com/channels/472484458856185878/472485684473626625/703753792243499038)_
+
+When you're done, press <kbd>CTRL+S</kbd> and <kbd>CTRL+X</kbd>. Your start script has now been created, and running `ls` should now show `start.sh`.
+
+To actually _run_ this script, you first will need to add executable permissions to the file.
+
+To do this, run:
+
+```shell script
+chmod +x start.sh
+```
+
+Now that the script is executable, we can run the proxy! Using the command:
+
+```
+./start.sh
+```
+
+will start up the Velocity jar. In the **rare** event that the proxy crashes, it will be automatically turned back on. If you wish to stop the proxy, pressing CTRL+C once will stop Velocity, and pressing CTRL+C again while it's not running will end the script.
+
+For now, end the script. We have one last thing to do.
+
+### Running Velocity in the Background
+
+As mentioned earlier, this is most likely a case where you'll want to run Velocity in the background.
+
+To do this, run this command:
+
+```shell script
+screen -S velocity ./start.sh
+```
+
+This will create a new `screen`, which will automatically run your Velocity script. Now that it's up and running, you can press <kbd>CTRL+A+D</kbd> to exit the screen, and now you can go about your day :)
+
+Remember, if you want to reconnect to your screen, use this command:
+
+```shell script
+screen -r velocity
+```
+
+# Extra Steps
+
+## Configuring Velocity
+
+It's great that Velocity is running, but now you need servers to connect to! Refer to the [Configuring Velocity guide found here](https://newsite-staging.velocitypowered.com/wiki/users/configuration/).
```

Comment:
Missed this, use a site absolute URL, i.e. `/wiki/users/configuration/`

Reply:
Done :)